packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -33,6 +33,8 @@ type TAccountTransferFormProps = {
 };
 
 const AccountOption = ({ account, idx }: TAccountsList) => {
+    const is_deriv_account = account.is_dxtrade || account.is_ctrader || account.is_mt || account.is_derivez;
```

Comment:
```suggestion
    const is_cfd_account = account.is_dxtrade || account.is_ctrader || account.is_mt || account.is_derivez;
```
Nit: Shouldn't these be CFD accounts instead of Deriv Accounts? 🤔

Reply:
@nijil-deriv yeap, your are right, my bad
fixing ...